user_docs/en/userGuide.t2t

```diff
@@ -346,6 +346,10 @@ When within a table, the following key commands are also available:
 | Move to last column | control+alt+end | Moves the system caret to the last column (staying in the same row) |
 | Move to first row | control+alt+pageUp | Moves the system caret to the first row (staying in the same column) |
 | Move to last row | control+alt+pageDown | Moves the system caret to the last row (staying in the same column) |
+| Speak column downwards | ``NVDA+control+alt+downArrow`` | Reads column vertically from the current table cell downwards to the last cell in the column. |
+| Speak row rightwards | ``NVDA+control+alt+rightArrow`` | Reads row horizontally from current table cell rightwards to the last cell in the row. |

```

Comment:
* Use "say all" in the name of the NVDA+control+alt+DownArrow/rightArrow commands, in order to use the same vocabulary as the "say all" command to reflect that the caret is moving while the text is being read. E.g.: "Say all in the current column" or shorter "Say all current column" or "say all in column".
* "Reads column/row" -> "Read the column/row": added "the" to avoid telegraphic style in the command description.
* "from current table cell " -> "from the current cell " (twice): drop "table" which is obvious and add "the" to avoid telegraphic style in the command description.

```suggestion
| Say all in column | ``NVDA+control+alt+downArrow`` | Reads the column vertically from the current cell downwards to the last cell in the column. |
| Say all in row | ``NVDA+control+alt+rightArrow`` | Reads the row horizontally from the current cell rightwards to the last cell in the row. |
```